src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -456,7 +457,7 @@ void VerifyV1ManifestContent(const Manifest& manifest, bool isSingleton, Manifes
     }
     else
     {
-        REQUIRE(manifest.Installers.size() == 2);
+        REQUIRE(manifest.Installers.size() == 3);

```

Comment:
>REQUIRE(manifest.Installers.size() == 3); [](http://example.com/codeflow?start=8&length=41)

This will fail the previous 1.0 and 1.1 tests

Reply:
Added logic to differentiate installer count check by manifest version